src/opm/output/eclipse/InteHEAD.cpp

```diff
@@ -343,10 +343,10 @@ enum index : std::vector<int>::size_type {
   ih_285       =      285       ,              //       0
   MAX_ANALYTIC_AQUIFERS= VI::intehead::MAX_ANALYTIC_AQUIFERS, //  Declared maximum number of analytic aquifers in model.  AQUDIMS(5).
   ih_287       =      287       ,              //       0
-  ih_288       =      288       ,              //       0
-  ih_289       =      289       ,              //       0
-  NOIUADS       =      VI::intehead::NO_IUADS,  //       0
-  NOIUAPS       =      VI::intehead::NO_IUAPS,  //       0
+  NUM_AQU_UDQS =      VI::intehead::NO_AQU_UDQS,      //       0
+  NUM_BLK_UDQS =      VI::intehead::NO_AQU_UDQS,      //       0
+  NUM_IUADS    =      VI::intehead::NO_IUADS,  //       0
+  NUM_IUAPS    =      VI::intehead::NO_IUAPS,  //       0
```

Comment:
These two lines look indentical. Is that on purpose?

Reply:
> These two lines look identical. Is that on purpose?

I don't quite see how these are identical.  Line 348 assigns `NUM_IUADS` (with a `D`) while line 349 assigns `NUM_IUAPS` (with a `P`).  Am I missing something?

Reply:
Indeed. I should clean my glasses. :flushed: Was wondering a bit whether a compiler would even allow that.